ykman-gui/qml/ChangePinView.qml

```diff
@@ -29,7 +29,8 @@ ColumnLayout {
     readonly property bool pinMatches: newPin.text === confirmPin.text
 
     function validPin() {
-        return (pinMatches) && (chosenPin.length >= minLength)
+        return (currentPin.length >= minLength)
```

Comment:
This blocks setting a FIDO2 PIN, since this will always be false in this view I believe.

Reply:
Oops, should be fixed now. (I don't have a FIDO2 device for testing with me right now...)